requirements.txt

```diff
@@ -0,0 +1,3 @@
+ansible
```

Comment:
I would require at least 2.7 here, as I like to use new powerful features ;-P

Reply:
Sounds good. Added :rocket: .